docs/shared/migration/adding-to-existing-project.md

```diff
@@ -23,120 +18,225 @@ You can make these scripts faster by leveraging Nx's caching capabilities. For e
 - You change some spec files: in that case the `build` task can be cached and doesn't have to re-run.
 - You update your docs, changing a couple of markdown files: then there's no need to re-run builds, tests, linting on your CI. All you might want to do is trigger the Docusaurus build.
 
-## Installing Nx on a Non-Monorepo Project
+## Install Nx on a Non-Monorepo Project
 
 Run the following command:
 
 ```shell
 npx nx@latest init
 ```
 
-This will
-
-- collect all the NPM scripts in the corresponding `package.json` files of your workspace packages
-- ask you which of those scripts are cacheable (e.g. build, test, lint)
-- ask you which of those scripts might need to be run in a certain order (e.g. if you run the `build` script you might want to first build all the dependent projects)
-- ask you for custom output folders that should be captured as part of the caching
-
-This process adds `nx` to your `package.json` at the root of your workspace:
-
-```json {% fileName="package.json" %}
-{
-  "name": "my-workspace",
-  ...
-  "devDependencies": {
-    ...
-    "nx": "17.2.0"
-  }
-}
-```
-
-In addition it generates a `nx.json` based on your answers during the setup process. This includes cacheable targets as well as some initial definition of the task pipeline. Here is an example:
-
-{% tabs %}
-{% tab label="Nx >= 17" %}
+This will set up Nx for you - updating the `package.json` file and creating a new `nx.json` file with Nx configuration based on your answers during the set up process. The set up process will suggest installing Nx plugins that might be useful based on your existing repository. The example below is using the `@nx/eslint` and `@nx/next` plugins to run ESLint and Next tasks with Nx:
 
 ```json {% fileName="nx.json" %}
 {
-  "targetDefaults": {
-    "build": {
-      "cache": true
+  "plugins": [
+    {
+      "plugin": "@nx/eslint/plugin",
+      "options": {
+        "targetName": "lint"
+      }
     },
-    "lint": {
-      "cache": true
+    {
+      "plugin": "@nx/next/plugin",
+      "options": {
+        "buildTargetName": "build",
+        "devTargetName": "dev",
+        "startTargetName": "start"
+      }
     }
-  }
+  ]
 }
 ```
 
-{% /tab %}
-{% tab label="Nx < 17" %}
+When Nx updates your `package.json` scripts, it looks for scripts that can be non-intrusively replaced with an Nx command that has caching automatically enabled. The `package.json` defined above would be updated to look like this:
```

Comment:
Maybe we shouldn't say "non-intrusively", just "scripts that can be replaced with an Nx command". It might make the operation seem weirder than it should be.